src/redux/books/Books.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Books = () => (
+
+  <form>
+    <input type="text" placeholder="Book Title" />
+    <input type="text" placeholder="Book Author" />
+    <button type="button">Add Book</button>
+  </form>
+
+);
+
```

Comment:
- [ ] As requested in the previous review this should take the Book title and author as props and render them. 
- [ ] It should also contain the remove book button that would just be a placeholder for now.
- [ ] In addition, please return this component to the components folder, the redux folder should only contain the redux store actions and reducers.